packages/web3/src/index.ts

```diff
@@ -364,3 +364,4 @@ export * as validator from 'web3-validator';
  */
 export * from 'web3-errors';
 export * from 'web3-types';
+export type { Web3Account, Wallet } from 'web3-eth-accounts';
```

Comment:
if there is any better place to export these, let me know!

Reply:
from web3-accts pkg

Reply:
all account types are exported from `web3-eth-accounts` but not from `web3`. See https://github.com/web3/web3.js/blob/7341-export-types/packages/web3-eth-accounts/src/index.ts#L42
How to solve that?

Reply:
I moved the export of these to packages/web3/src/types.ts